motr/m0crate/crate.c

```diff
@@ -224,6 +224,16 @@ int workload_init(struct workload *w, enum cr_workload_type wtype)
         w->cw_nr_dir    = cr_default_nr_dir;
         w->cw_read_frac = cr_default_read_frac;
 
+	/* set default values before yaml parsing */
+	if (wtype == CWT_INDEX)
+	{
```

Comment:
According to "doc/coding-style", we put the opening brace at the end of condition line:
```
if (wtype == CWT_INDEX) {
```

Reply:
please fix similar issues in other places

Reply:
Fixed.